test.js

```diff
@@ -2,7 +2,6 @@ import test from 'ava';
 import chalk from 'chalk';
 import execa from 'execa';
 
-process.env.FORCE_COLOR = true;
```

Comment:
Curious, what's the reason for removing this?

Reply:
Oh good question. I think it's not necessary anymore, after adding 6458c59ddfe584819b195890974f6c196317b1b8:

```diff
--- a/package.json
+++ b/package.json
@@ -16,7 +16,7 @@
                "node": ">=6"
        },
        "scripts": {
-               "test": "xo && ava"
+               "test": "xo && FORCE_COLOR=1 ava"
        },
        "files": [
                "cli.js"
```

Basically setting `process.env.FORCE_COLOR` wasn't really doing anything anymore, because it happens after the import of `chalk` and that's too late. For that reason, I moved it to `package.json`.